lib/ex_aws/ses.ex

```diff
@@ -355,6 +375,28 @@ defmodule ExAws.SES do
     request(:create_template, params)
   end
 
+  @doc """
+  Create an email template via V2 API. See https://docs.aws.amazon.com/ses/latest/APIReference-V2/API_CreateEmailTemplate.html
+  """
+  @spec create_email_template(String.t(), String.t(), String.t(), String.t()) :: ExAws.Operation.JSON.t()
+  def create_email_template(template_name, subject, html, text) do
+    template_content =
+      %{
+        "Subject" => subject
+      }
+      |> put_if_not_nil("Html", html)
+      |> put_if_not_nil("Text", text)
+
+    params =
+      %{
+        "TemplateName" => template_name
+      }
+      |> Map.put("TemplateContent", template_content)
```

Comment:
I don't see a sense to use two different methods to add elements to a map. 

Can the same thing be accomplished with: 
```suggestion
    params =
      %{
        "TemplateName" => template_name,
        "TemplateContent" => template_content
      }
```

Reply:
I agree.